pyart/correct/__init__.py

```diff
@@ -43,5 +45,6 @@
 from ..filters.gatefilter import GateFilter, moment_based_gate_filter
 from .unwrap import dealias_unwrap_phase
 from .region_dealias import dealias_region_based
+from .despeckle import find_objects, despeckle
```

Comment:
There are some issues if a module and function have the same name. Can the module be renamed (maybe despecking) or the function (despeckle_field)?